cadl-tests/src/test/java/com/models/property/types/DurationOperationClientTest.java

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.models.property.types;
+
+import com.models.property.types.models.DurationProperty;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+class DurationOperationClientTest {
+
+    DurationOperationClient client = new DurationOperationClientBuilder().buildClient();
+
+    @Disabled("cadl-ranch definition error")
```

Comment:
These two tests are disabled, because I found an issue from cadl-ranch.
I raised the issue here: https://github.com/Azure/cadl-ranch/issues/90
I tried to fix the issue and create a pr for cadl-ranch: https://github.com/Azure/cadl-ranch/pull/89